third_party/java_deps/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# === This BUILD file is auto-generated, modify the GnBuildGenerator task instead of editing this file manually. ===
+
+import("//build_overrides/chip.gni")
+import("${chip_root}/build/chip/java/rules.gni")
+
+java_prebuilt("annotation") {
+  jar_path = "artifacts/jsr305-3.0.2.jar"
```

Comment:
please force check-in jsr305 jar file

Reply:
can we grab it somehow from maven/cipd to not check in binary files.

Reply:
Acked